week-2-wide-perspective/w02e02-data-manipulation/src/observers/ordersObserver.js

```diff
@@ -0,0 +1,80 @@
+import { addOrder, getOrders } from "../services/http.service.js";
```

Comment:
Cały ten plik można by było sobie zrefaktoryzować i wydzielić odpowiedzialności.

Przygotowując generycznie Twoją koncepcję z `Observable`.
Jednakże rozumiem sprawę, i przyjmuje Twoje @kubaGalinski tłumaczenie z komentarza do PR 😄.

Jeśli znajdziesz czas / chęci.
To tutaj widzę "room for improvements" i generalnie miejsce warte refaktoringu.

Jednakże - super fajny ten `Observable`.
Fajnie go sobie "od zera" wykraftowałeś.
Tutaj pomocna by była ta biblioteka `events` dorzucona w `index.html`
```html
<script type="text/javascript" src="https://unpkg.com/events@3.3.0/events.js"></script> 
```

bo mógłbyś skorzystać z właśnie tamtego API do zrobienia zadań które zdecydowałeś się tutaj przygotować "z ręki".

Super 🥇 !